prime-router/metadata/HL7/catchall/hl7/datatypes/EI/Device-udiCarrier.yml

```diff
@@ -8,6 +8,11 @@ deviceIdentifier:
 extension:
   expressionType: nested
   generateList: true
+  vars:
+    ei2: EI.2
+    ei3: EI.3
+    ei4: EI.4
+  condition: $ei2 NOT_NULL || $ei3 NOT_NULL || $ei4 NOT_NULL
```

Comment:
This and many other changes to mappings address an issue where resources that have URLs but no value were created. This is prevented by proactively checking if the value exists.